tutorial/README.md

```diff
@@ -0,0 +1,42 @@
+# Why Helm?
+
+[Helm](https://docs.helm.sh/) is often described as the Kubernetes application package manager. So, what does Helm give you over using Kubernetes directly?
+
+# Objectives
+
+These labs provide an insight on the advantages of using Helm instead of using Kubernetes directly through `kubectl`. By the end of this course, you'll:
```

Comment:
``` These labs provide an insight on the advantages of using Helm over using Kubernetes directly through `kubectl`. When you complete all the labs, you'll```
(We will eventually make these labs into courses, but until then, let's just call them labs.)

Reply:
done